micro-rdk-installer/src/main.rs

```diff
@@ -238,14 +252,31 @@ fn monitor_esp32(baud_rate: Option<u32>) -> Result<(), Error> {
         no_stub: false,
     };
     let conf = Config::default();
+    log::info!("Connecting...");
     let flasher = connect(&connect_args, &conf).map_err(|_| Error::FlashConnect)?;
     let pid = flasher.get_usb_pid().map_err(Error::EspFlashError)?;
+    log::info!("Connected. Starting monitor...");
     monitor(flasher.into_interface(), None, pid, 115_200)
         .map_err(|err| Error::MonitorError(err.to_string()))?;
     Ok(())
 }
 
+fn init_logger() {
+    let env = env_logger::Env::default()
+        .filter_or("MY_LOG_LEVEL", "info")
```

Comment:
are users going to set the log level with `MY_LOG_LEVEL` instead of `RUST_LOG` ?

Reply:
I actually think this should always be at info? Is there any reason to have it customizable?